src/main/resources/assets/edenring/lang/book/en_us.json

```diff
@@ -43,11 +43,11 @@
 						"§3Mass: §828.5 Em",
 						"§3Class: §8Ice Giant",
 						"§3Atmosphere:",
-						"§3  §862% N2",
-						"§3  §818% H2O",
-						"§3  §812% NH3",
-						"§3  §805% H2",
-						"§3  §803% Other",
+						"§3  §862% N\u2082",
+						"§3  §818% H\u2082O",
+						"§3  §812% NH\u2083",
+						"§3  §80.5% H\u2082",
```

Comment:
Dots here are not needed

Reply:
No, they are needed, because without them it is displayed as 05

Reply:
Yes, 05 is designed value